c_common/models/compressors/src/sorter/bit_field_reader.h

```diff
@@ -150,6 +150,30 @@ static inline void print_structs(
     }
 }
 
+static uint32_t n_redundant(filter_info_t *filter) {
```

Comment:
so this is bemusing. line 39 detect_redundant_packet_count is doing the same thing as this. and line 39 is more doced. but yours is a better impl. i recommend merging them into 1

Reply:
Merged